index.html

```diff
@@ -90,7 +90,7 @@ <h1 class="noMobile">
     <div>
       רוצים לתת חסות? צרו קשר עם <a href="mailto:board@hamakor.org.il">board@hamakor.org.il</a>
     </div>
-    <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScQwZkPa0lLdbFEG5Ek3XuzEd2oeoe6Ca_bOCP4aWp_0npZRw/viewform" width="640" height="2500"></iframe>
+    <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScQwZkPa0lLdbFEG5Ek3XuzEd2oeoe6Ca_bOCP4aWp_0npZRw/viewform" width="480" height="1600"></iframe>
```

Comment:
On my phone at least, in "portrait mode", the form is still cut off to the side (too wide), so does not improve the visibility.
And in desktop mode the original width looks much better.

(on phone "landscape mode" (rotated to the side), both look OK)